adafruit_turtle.py

```diff
@@ -94,8 +98,11 @@ class Vec2D(tuple):
     #     k*a and a*k multiplication with scalar
     #     |a| absolute value of a
     #     a.rotate(angle) rotation
-    def __init__(self, x, y):
-        super().__init__((x, y))
+    def __new__(cls, x, y):
+        return (x, y)
+
+    def __getitem__(self, index):
+        return getattr(self, index)
```

Comment:
The way you overrode `__getitem__`  here gave me inspiration for a different approach. If we can have Vec2D "hold" a tuple instead of "being" a tuple then perhaps we can have the "best of both worlds" and it will work on both microcontroller and SBC / CPYthon.